pkg/acs/calacs/lib/getacskeys.c

```diff
@@ -46,14 +46,23 @@ int GetACSKeys (ACSInfo *acs, Hdr *phdr) {
   
 	if (GetKeyStr (phdr, "ROOTNAME", NO_DEFAULT, "",
                  acs->rootname, ACS_CBUF))
-    return (status);
+       return (status);
   
 	if (GetKeyStr (phdr, "APERTURE", USE_DEFAULT, "", acs->aperture, ACS_CBUF))
-    return (status);
+       return (status);
+    {unsigned int i;
+       for (i=0; i < strlen(acs->aperture); i++) {
+           acs->aperture[i] = toupper (acs->aperture[i]);
```

Comment:
This is only acceptable/valid if it is "illegal" for the char set to be [a-z], if not it must be removed. Even if it is I really don't think it is this function's responsibility to do so.

Reply:
Imagine someone wanting to write a validity wrapper to ``GetACSKeys()``, they now can't because the functionality/semantics has been mangled.